lib/wit/main.py

```diff
@@ -123,13 +123,13 @@ def foreach(ws, args):
 
         log.info("Entering '{}'".format(pkg.name))
 
+        command = " ".join([args.cmd] + args.args)
         location = str(ws.root / pkg.name)
-        proc = subprocess.run([args.cmd, *args.args], shell=True,
+        proc = subprocess.run(command, shell=True,
                               env=env, cwd=location, universal_newlines=True)
```

Comment:
Does this work when the foreached command itself has quotes? eg.

```
$ wit foreach git grep 'Hello World'
```

Reply:
Might be good to add a test case for this.

Reply:
good catch